inspirehep/modules/literaturesuggest/forms.py

```diff
@@ -436,7 +424,7 @@ def _is_other_language(language):
         label=_('Date of Submission'),
         description='Format: YYYY-MM-DD, YYYY-MM or YYYY.',
         validators=[date_validator],
```

Comment:
Note also that the commit subject is incorrect: the validator is still there, what was removed was the widget, which was completing the incomplete dates.